realsense_ros_camera/src/base_realsense_node.cpp

```diff
@@ -392,7 +392,7 @@ void BaseRealSenseNode::setupStreams()
                 publishFrame(frame, t);
             }
 
-            if(_pointcloud && is_depth_frame_arrived && is_color_frame_arrived &&
+            if(_pointcloud && is_depth_frame_arrived && //is_color_frame_arrived &&
```

Comment:
Why is this commented out? Shouldn't it be removed completely?

It'd help to understand what's going on here if you explain what is `is_color_frame_arrived` used for and especially same for `_enable[COLOR]`

Reply:
This should likely be

if(_pointcloud && is_depth_frame_arrived && (!_enable[COLOR] || is_color_frame_arrived) &&